src/app/common/DataOverview.vue

```diff
@@ -173,28 +173,6 @@
               size="20"
             />
           </template>
-
-          <template
```

Comment:
**Comment**: For the time being, I’m removing the details column as we’re only using it for data plane proxies.

We might consider re-introducing this for all list views, but let’s discuss this separately.

Reply:
Cool yeah sounds good, we should add that to our next GUI sync